### PR TITLE
Basic editor

### LINKED ISSUE #17: Create Killer Sudoku Editor
- [x] develop editor frontend
  - [x] #13
  - [x] #14
  - [x] #15
  - [x] #16
- [x] setup API for accepting grid
  - [x] #18 
  - [x] #19 
- [x] setup API for starting corresponding WebSocket Session
  - [x] #20 
  - [x] #21

### PR DESCRIPTION
**Pull request by Jacky-Ickx**
closes #17

### CODE REVIEW

File: backend/app/build.gradle [hunk 1]
```diff
@@ -32,15 +34,38 @@ repositories {
...
     // Use JUnit Jupiter for testing.
     testImplementation 'org.junit.jupiter:junit-jupiter:5.9.1'
     testImplementation 'org.apache.commons:commons-collections4:4.4'
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
 
     // This dependency is used by the application.
     implementation 'com.google.guava:guava:31.1-jre'
+    implementation 'org.javatuples:javatuples:1.2'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
```
Comment (Y05h129) [inline backend/app/build.gradle:59]: indentation

File: frontend/src/app/core/services/sudoku.service.ts
Comment (Y05h129) [inline frontend/src/app/core/services/sudoku.service.ts:1]: indetation

File: frontend/src/app/shared/cell/_cell.component-theme.scss
Comment (Y05h129) [inline frontend/src/app/shared/cell/_cell.component-theme.scss:1]: indetation

File: frontend/src/app/shared/grid/grid.component.ts
Comment (Y05h129) [inline frontend/src/app/shared/grid/grid.component.ts:1]: indetation

**Review by Y05h129 — APPROVED** [review]
@Jacky-Ickx docstrings are required, but can be ignored in this case.
See Ticket #26

File: backend/app/build.gradle (hunk 1 above)
Comment (Jacky-Ickx) [reply backend/app/build.gradle:59]: fcd767a4826205dec1cfe4e5cbdd35cf6240429f

Comment (Jacky-Ickx) [reply backend/app/build.gradle:59]: fcd767a4826205dec1cfe4e5cbdd35cf6240429f

File: frontend/src/app/core/services/sudoku.service.ts
Comment (Jacky-Ickx) [reply frontend/src/app/core/services/sudoku.service.ts:1]: fcd767a4826205dec1cfe4e5cbdd35cf6240429f

File: frontend/src/app/shared/cell/_cell.component-theme.scss
Comment (Jacky-Ickx) [reply frontend/src/app/shared/cell/_cell.component-theme.scss:1]: fcd767a4826205dec1cfe4e5cbdd35cf6240429f

File: frontend/src/app/shared/grid/grid.component.ts
Comment (Jacky-Ickx) [reply frontend/src/app/shared/grid/grid.component.ts:1]: fcd767a4826205dec1cfe4e5cbdd35cf6240429f

**Pull request merged**